src/util/parseAlgoliaHit.js

```diff
@@ -39,7 +39,10 @@ function parseHighlightedAttribute({ preTag, postTag, highlightedValue = '' }) {
 
       if (splitByPostTag[1] !== '') {
         elements.push({
-          value: splitByPostTag[1],
+          // Vue removes nodes which are just a single space (vuejs/vue#9208),
```

Comment:
is this what you expected @francoischalifour?

Reply:
Yep, thank you!